lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -491,7 +489,7 @@ export class UserAgentApplication {
       }
     }
 
-    acquireTokenAuthority.resolveEndpointsAsync().then(() => {
+    acquireTokenAuthority.resolveEndpointsAsync().then(async () => {
```

Comment:
nit: prefer not mixing async and Promise.then().
`acquireTokenAuthority.resolveEndpointAsync().then(() => {
....
return this.pollWindowForHash(popUpWindow);
}).then(hash => {
...
});`
Functionality they are the same but consistency IMO